src/main/java/org/tron/walletcli/Client.java

```diff
@@ -4403,6 +4409,47 @@ private void getChainParameters() {
     }
   }
 
+  private void getBlockByIdOrNum(String[] parameters) {
+    String idOrNum = null;
+    boolean detail = false;
+    if (parameters == null || parameters.length == 0) {
+      // query current header
+      System.out.println("Get current header !!!");
```

Comment:
if no parameters, it should remind how to use the command.
such as "GetBlockByIdOrNum needs 2 parameters like following: "
...